### PR TITLE
Run prelude.rs after init.evcxr if one exists, and support specifying config path via environment.

### LINKED ISSUE #79: Support loading a prelude file?
As I mentioned in https://github.com/google/evcxr/issues/67#issuecomment-562889010, it's a bit annoying to 

Currently one option would be for me to define this in a separate crate (say 'thom_evcxr_prelude'), use `:dep thom_evcxr_prelude { path = "/Users/thom/src/thom_evcxr_prelude }` followed by `use thom_evcxr_prelude::*`.

However, when I tried this it impacted startup time of evcxr noticeably, and so I chose to just put everything on one line, since I start evcxr much more often than I touch that file. Defining a separate crate also feels like overkill if you have just, say, a small number of declarations and use statements which might be too involved to conveniently fit on one line (the initial suggesion in https://github.com/google/evcxr/issues/67#issuecomment-553833421 for example)

An easy solution would be just to have `<evcxr config dir>/prelude.rs` which is loaded as a single rust expression after init.evcxr runs (doing it after allows init.evcxr to include a dep, which the prelude uses, for example).

Alternatively, a `:load <path/to/blah.rs>` might be a more general useful feature, and would allow splitting to multiple files, and solves the 'when should it run' question, but seems like it might confuse users who try to point it at a lib.rs or mod.rs or something, and have it cause problems that way.

I would be happy to submit a PR for this if it would be accepted.

### PR DESCRIPTION
**Pull request by thomcc**
Tested manually (both with and without the prelude), since there aren't automated tests for this I could find.

### CODE REVIEW

**Comment by thomcc** [comment]
Ah, I missed that you said you'd accept #78, one minute and I'll add a commit for that too (since otherwise it would conflict).

**Comment by thomcc** [comment]
Note: in case it matters, I tested with evcxr_repl, I'm not sure how to set up the jupyter stuff to test evcxr_jupyter, but I have no reason to think it wouldn't work (but I am willing to try if you think it's needed).

File: evcxr/src/lib.rs [hunk 1]
```diff
@@ -39,3 +39,14 @@ pub use crate::command_context::CommandContext;
 pub use crate::errors::{CompilationError, Error};
 pub use crate::eval_context::{EvalContext, EvalContextOutputs, EvalOutputs};
 pub use crate::runtime::runtime_hook;
+
```
Comment (thomcc) [inline evcxr/src/lib.rs]: Wasn't sure what the best place for this function was. `lib.rs` is mostly just re-exports, but nothing else seemed that relevant, and it didn't seem worth adding a new module just for this function. Let me know if you'd like me to move or change this somehow though.

**Comment by davidlattimore** [comment]
Thanks!

**Pull request merged**